### ISSUE TITLE
How do I enable HTML reports for FindBugs?

### ISSUE THREAD

**Issue by tony19** [novoda/gradle-static-analysis-plugin#74]
I tried the following to no avail:

```
staticAnalysis {
  findbugs {
    reports {
      xml.enabled false
      html.enabled true
    }
  }
}
```
and
```
tasks.withType(FindBugs) {
  reports {
    xml.enabled false
    html.enabled true
  }
}
```

**Comment by mr-archano** [comment]
Hi @tony19, as you might now findbugs supports only one type of report at time. To avoid this issue we override the configuration and enable XML reports while we manually add a task that generates an HTML from it (see [`FindbugsConfigurator.groovy#L130-L132](https://github.com/novoda/gradle-static-analysis-plugin/blob/master/plugin/src/main/groovy/com/novoda/staticanalysis/internal/findbugs/FindbugsConfigurator.groovy#L130-L132) and [`GenerateFindBugsHtmlReport.groovy`](https://github.com/novoda/gradle-static-analysis-plugin/blob/master/plugin/src/main/groovy/com/novoda/staticanalysis/internal/findbugs/GenerateFindBugsHtmlReport.groovy)).
Basically just running `check` you should be able to find an HTML report in the same folder of the the XML report. Can you check please and let me know?

**Comment by tony19** [comment]
Thanks. I was running into a misleading Checkstyle error with `gradle check` (caused by a missing `checkstyle.xml` config), so I didn't see the html report for FindBugs.
